src/Mpociot/BotMan/Drivers/SlackDriver.php

```diff
@@ -124,7 +125,7 @@ public function reply($message, $matchingMessage, $additionalParameters = [])
         if (! Collection::make($matchingMessage->getPayload())->has('team_domain')) {
             $this->replyWithToken($message, $matchingMessage, $additionalParameters);
         } elseif ($this->isSlashCommand()) {
```

Comment:
@mpociot While sending the PR to dirver-slack just realized we don't need this else lol